proxy/http/remap/RemapPlugins.h

```diff
@@ -75,6 +79,7 @@ struct RemapPlugins : public Continuation {
 
 private:
   unsigned int _cur        = 0;
+  unsigned int _rewriten   = 0;
```

Comment:
This is minor, but could we name the member `_rewritten` with two `t`s?

Reply:
I missed that :)